### DISCUSSION TITLE
[Playground] HMR support possible?

### DISCUSSION THREAD

**Discussion by Innei** [toeverything/blocksuite#1044] [Q&A]
Now no matter where to modify the code will make the page reload, lost page state, development experience is not good. Does playground moving to React make HMR support possible?

**Comment by lawvs** [comment]
In fact, the playground has HMR, but it has a limited effect.

See https://github.com/toeverything/blocksuite/pull/621

**Comment by lawvs** [comment]
You can use the share URL button to keep the page state.


https://user-images.githubusercontent.com/18554747/217169242-afc7a26c-b971-4b2f-a84a-2246fffed5de.mov

**Comment by Innei** [comment]
I see, but it only work on `'../blocks/src/**/*'`, and if modify other code, it always reload page. By the way, playground dev:hmr mode is not default in dev script in root workspace.
<img width="894" alt="image" src="https://user-images.githubusercontent.com/41265413/217174467-468ca519-6c35-459c-8ba3-4aab7164ef13.png">

Can modify root dev script to start playground dev:hmr?
